### DISCUSSION TITLE
Quickly return to the top from the Explore page

### DISCUSSION THREAD

**Discussion by fanenr** [community/community#64404] [Mobile]
# Question

There's no way to quickly return to the top of the Explore page when I've browsed through many items on the Explore page.

# Suggestion

Maybe I can click the explore icon repeatedly to quickly go back to the top, or add a button?

**Comment by Iron-Ham** [comment]
Hi @fanenr, 

Thank you for your feedback -- it's what helps us make the GitHub Mobile experience better.
If you could provide us with some additional information: What version of the app are you on, and are you on the iOS or Android version? 


Thank you! 
Hesham

**Comment by fanenr** [comment]
> What version of the app are you on, and are you on the iOS or Android version? 

Sorry about that.
I'm using GitHub version 1.126.0.
The OS is Android 13.
